modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/assets/components/AssetAppearsOnCard.tsx

```diff
@@ -29,6 +29,15 @@ export enum Accessor {
 	PreviewsMetric = 'previewsMetric'
 }
 
+export enum EmptyStateLink {
+	Asset = URLConstants.AssetsCustomAssetsListDocumentation,
+	Blog = URLConstants.AssetsAppearsBlogsOnDocumentation,
+	Document = URLConstants.AssetsAppearsDocumentsAndMediaOnDocumentation,
+	Form = URLConstants.AssetsAppearsFormsOnDocumentation,
+	Journal = URLConstants.AssetsAppearsWebContentOnDocumentation,
+	WebContent = URLConstants.AssetsAppearsWebContentOnDocumentation
```

Comment:
Hi @rafaellajordao these two constants are the same, could you remove `journal`?

Reply:
removed!